Description: Pass LDFLAGS when linking test binary
Author: Michael Hudson-Doyle <michael.hudson@ubuntu.com>
Origin: vendor
Bug: https://llvm.org/bugs/show_bug.cgi?id=27597
Last-Update: 2016-05-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/tsan/go/buildgo.sh
+++ b/lib/tsan/go/buildgo.sh
@@ -125,7 +125,7 @@
 fi
 $CC $DIR/gotsan.cc -c -o $DIR/race_$SUFFIX.syso $FLAGS $CFLAGS
 
-$CC $OSCFLAGS test.c $DIR/race_$SUFFIX.syso -m64 -g -o $DIR/test $OSLDFLAGS
+$CC $OSCFLAGS test.c $DIR/race_$SUFFIX.syso -m64 -g -o $DIR/test $OSLDFLAGS $LDFLAGS
 
 export GORACE="exitcode=0 atexit_sleep_ms=0"
 if [ "$SILENT" != "1" ]; then
